### PR TITLE
{bio}[gompi/2022a] HyPhy v2.5.60

### PR DESCRIPTION
**Pull request by WilleBell**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by WilleBell** [comment]
Test report by @WilleBell
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node306.hydra.os - Linux Rocky Linux 8.8, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 3.6.8
See https://gist.github.com/WilleBell/86dc1422ed5e1cd0a4354c403b583e82 for a full test report.

File: easybuild/easyconfigs/h/HyPhy/HyPhy-2.5.25-gompi-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+easyblock = "CMakeMake"
+
+name = 'HyPhy'
+version = '2.5.25'
```
Comment (Micket) [inline easybuild/easyconfigs/h/HyPhy/HyPhy-2.5.25-gompi-2022a.eb:4]: Why such an old version? It's even older than the gompi-2021a variant we already have. 2.5.60 surely?

Comment (verdurin) [reply easybuild/easyconfigs/h/HyPhy/HyPhy-2.5.25-gompi-2022a.eb:4]: @WilleBell any thoughts on the suggestion from @Micket ?

Comment (WilleBell) [reply easybuild/easyconfigs/h/HyPhy/HyPhy-2.5.25-gompi-2022a.eb:4]: We had a request for this specific version (for some reason I don't really know). But if you think it is too old for this toolchain I can upgrade the version (or downgrade the toolchain).

Comment (verdurin) [reply easybuild/easyconfigs/h/HyPhy/HyPhy-2.5.25-gompi-2022a.eb:4]: You might want to keep the specific older version for a local repo, and upgrade the version here to the latest one?
That way you meet the request of your user and keep the upstream repo up to date?

Comment (WilleBell) [reply easybuild/easyconfigs/h/HyPhy/HyPhy-2.5.25-gompi-2022a.eb]: Makes sense

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/41970405c2a81cc823d8782cfd798be4 for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
@boegelbot please test @ jsc-zen3

**Comment by verdurin** [comment]
Going in, thanks @WilleBell!

**Comment by verdurin** [comment]
Going in, thanks @WilleBell!

**Pull request merged**